src/modules/RokulRunnings.ts

```diff
@@ -175,13 +230,45 @@ export class RokulRunnings {
     delayInMillis = this.pressDelayInMillis,
     retries = this.retries,
     params,
+    keyType = 'press',
   }: {
     sequence: Buttons[] | string[];
     delayInMillis?: number;
     retries?: number;
     params?: Params;
+    keyType?: 'up' | 'down' | 'press';
+  }) {
+    const newSequence: { up?: string | Buttons; down?: string | Buttons; press?: string | Buttons }[] = [];
+    sequence.forEach(button => {
+      if (!newSequence) {
```

Comment:
I don't think this condition will ever be met

Reply:
Ahh I think you're right. Would this be better as...
`if (!newSequence[0])`?

Reply:
probably a length check === 0

Reply:
wait, it will always be an empty array when you start so I think you can just always `push`.

Reply:
I was thinking that if I had initialized it but it didn't have a size the `.push` would fail?

Reply:
nope, if you create an empty array you can use any of the methods from the Array prototype.

Reply:
Resolved in 25ff9b6